platform/viewer/src/googleCloud/api/GoogleCloudApi.js

```diff
@@ -22,6 +22,13 @@ class GoogleCloudApi {
     return this.urlBase + `/projects`;
   }
 
+  getUrlBaseDicomWeb(project, location, dataset, dicomstore) {
+    return (
+      this.urlBase +
+      `/projects/${project}/locations/${location}/datasets/${dataset}/dicomStores/${dicomstore}/dicomWeb`
```

Comment:
For me this PR is good, but I would remove the`/dicomWeb` at the end. Thoughts @pieper?

Reply:
I agree, no need for `dicomWeb`.

Otherwise yes, LGTM 👍

Reply:
Ok removed it..